obspy/clients/filesystem/tests/test_sds.py

```diff
@@ -222,6 +222,39 @@ def test_sds_report(self):
             if failed:
                 raise Exception
 
+    def test_get_waveforms_bulk(self):
+        """
+        Test get_waveforms_bulk method.
+        """
+        for year, doy in ((2015, 213), (2015, 1), (2012, 1)):
+            t = UTCDateTime("%d-%03dT00:00:00" % (year, doy))
+            with TemporarySDSDirectory(year=year, doy=doy) as temp_sds:
+                chunks = [
+                    ["AB", "XYZ", "", "HHZ", t,
+                    t + 20, os.path.join(temp_sds.tempdir, "file_1.mseed")],
+                    ["AB", "XYZ", "", "HHN", t + 20,
+                    t + 40, os.path.join(temp_sds.tempdir, "file_2.mseed")],
+                    ["AB", "XYZ", "", "HHE", t + 40,
+                    t + 60, os.path.join(temp_sds.tempdir, "file_3.mseed")],
+                    ["CD", "ZZZ3", "00", "BHZ", t + 60,
+                    t + 80, os.path.join(temp_sds.tempdir, "file_4.mseed")],
+                    ["CD", "ZZZ3", "00", "BHN", t + 80,
+                    t + 100, os.path.join(temp_sds.tempdir, "file_5.mseed")],
+                    ["CD", "ZZZ3", "00", "BHE", t + 120,
+                    t + 140, os.path.join(temp_sds.tempdir, "file_6.mseed")]
+                ]
+                client = Client(temp_sds.tempdir)
+                ret_val = client.get_waveforms_bulk(chunks)
+                contents = [("file_1.mseed", "AB.XYZ..HHZ"),
+                        ("file_2.mseed", "AB.XYZ..HHN"),
+                        ("file_3.mseed", "AB.XYZ..HHE"),
+                        ("file_4.mseed", "CD.ZZZ3.00.BHZ"),
+                        ("file_5.mseed", "CD.ZZZ3.00.BHN"),
+                        ("file_6.mseed", "CD.ZZZ3.00.BHE")]
+                self.assertEqual(ret_val,
+                             sorted([os.path.join(temp_sds.tempdir, _i[0])
+                                     for _i in contents]))
```

Comment:
This testing for file paths is weird.. the output of `get_waveforms_bulk()` should be a `Stream` object, and all you want to do is check the contents of the stream if it has all expected `Trace`s in it.

Reply:
Thank you for the feedback.  I will change the UT to check for the correct traces.

Reply:
Thank you, this insight sparked enough such that I have a working test that I believe properly tests this issue.  Will push it again within the next few hours.

CC @dsentinel